tests/raft_scenarios/fancy_election.1

```diff
@@ -61,7 +61,11 @@ dispatch_one,1
 # Node 2 responds in support
 dispatch_one,2
 
+# Node 1 appends a new committable entry to confirm its primaryship
+replicate,2,ImTheCaptainNow
+
 # Node 1 sends an initial AppendEntries probe
+periodic_one,1,10
```

Comment:
The change here is semantically the same as in `scenarios_gen.py`. Scenarios that previously reached state sync now need an additional write in the new term and an additional round-trip to confirm commit of this.